### PR TITLE
Added homoglyph functionality to visual_similiar_chars.

### PR DESCRIPTION
**Pull request by shashank-m**
Replaces characters with their homoglyph values from dictionary/
example key value pair in dictionary:
d={"a": "@aɑαа⍺ａ𝐚𝑎𝒂𝒶𝓪𝔞𝕒𝖆𝖺𝗮𝘢𝙖𝚊𝛂𝛼𝜶𝝰𝞪"}
Any one of the homoglyph is chosen at random for each char.

### CODE REVIEW

File: decepticonlp/transforms/perturb.py [hunk 1]
```diff
@@ -232,16 +237,84 @@ def visual_similar_chars(word, *arg, ignore=True):
...
-    int_pick = np.random.randint(0, high=unicode_array.shape[0], size=len(word))
+    if arg[method_pick] == "unicode":
+        char_array = np.array(list(word))
 
-    picked_unicode = unicode_array[int_pick]
+        int_pick = np.random.randint(0, high=unicode_array.shape[0], size=len(word))
 
-    perturbed_array = np.char.add(char_array, picked_unicode)
-    return "".join(perturbed_array)
+        picked_unicode = unicode_array[int_pick]
```
Comment (Sharad24) [inline decepticonlp/transforms/perturb.py:301]: Use `np.random.choice`?

File: decepticonlp/transforms/perturb.py [hunk 2]
```diff
@@ -232,16 +237,84 @@ def visual_similar_chars(word, *arg, ignore=True):
...
+    elif arg[method_pick] == "homoglyph":
+        char_list = list(word)
+        homoglyph_char_list = []
+        for char in char_list:
+            try:
+                glyph_string = d[char]
+                glyph_pick = np.random.choice(len(glyph_string), 1)[0]
+                homoglyph_char_list.append(glyph_string[glyph_pick])
+            except KeyError:
+                homoglyph_char_list.append(char)
```
Comment (Sharad24) [inline decepticonlp/transforms/perturb.py:314]: Also print out a warning as to why it is going into exception

File: decepticonlp/transforms/perturb.py (hunk 1 above)
Comment (shashank-m) [reply decepticonlp/transforms/perturb.py:301]: Yeah sorry forgot about that.

File: decepticonlp/transforms/perturb.py (hunk 2 above)
Comment (shashank-m) [reply decepticonlp/transforms/perturb.py:314]: hmm cool

File: decepticonlp/transforms/perturb.py [hunk 3]
```diff
@@ -232,16 +237,84 @@ def visual_similar_chars(word, *arg, ignore=True):
...
     )
+    d = {
+        "a": "@aɑαа⍺ａ𝐚𝑎𝒂𝒶𝓪𝔞𝕒𝖆𝖺𝗮𝘢𝙖𝚊𝛂𝛼𝜶𝝰𝞪",
+        "b": "bƄЬᏏᑲᖯｂ𝐛𝑏𝒃𝒷𝓫𝔟𝕓𝖇𝖻𝗯𝘣𝙗𝚋",
+        "c": "cϲсᴄⅽⲥꮯｃ𐐽𝐜𝑐𝒄𝒸𝓬𝔠𝕔𝖈𝖼𝗰𝘤𝙘𝚌",
+        "d": "dԁᏧᑯⅆⅾꓒｄ𝐝𝑑𝒅𝒹𝓭𝔡𝕕𝖉𝖽𝗱𝘥𝙙𝚍",
+        "e": "eеҽ℮ℯⅇꬲｅ𝐞𝑒𝒆𝓮𝔢𝕖𝖊𝖾𝗲𝘦𝙚𝚎",
+        "f": "fſϝքẝꞙꬵｆ𝐟𝑓𝒇𝒻𝓯𝔣𝕗𝖋𝖿𝗳𝘧𝙛𝚏𝟋",
+        "g": "gƍɡցᶃℊｇ𝐠𝑔𝒈𝓰𝔤𝕘𝖌𝗀𝗴𝘨𝙜𝚐",
+        "h": "hһհᏂℎｈ𝐡𝒉𝒽𝓱𝔥𝕙𝖍𝗁𝗵𝘩𝙝𝚑",
```
Comment (rohts-patil) [inline decepticonlp/transforms/perturb.py:248]: I think writing a huge static dictionary in the method is not good. This should be written in another py file and imported here. @shashank-m

Comment (shashank-m) [reply decepticonlp/transforms/perturb.py:248]: Yeah I'm doing that now

**Pull request closed without merging**